Khan/alertlib
# Add JIRA info for security team.

## Summary:
Add mapping to Security JIRA project so tickets can be filed.

Issue: https://khanacademy.atlassian.net/browse/SEC-19

## Test plan:
- run `python3 alert.py --bugtracker=Security`
- ensure a ticket is successfully created

Created ticket: https://khanacademy.atlassian.net/browse/SEC-164

Screenshot of created ticket (in case the ticket gets deleted):

![image](https://github.com/user-attachments/assets/7134cd89-d80c-4d80-a3b6-27adc4dc53e2)
